backend/inspirehep/records/api/data.py

```diff
@@ -16,22 +16,5 @@ class DataRecord(CitationMixin, InspireRecord):
     pid_type = "dat"
     pidstore_handler = PidStoreData
 
-    @classmethod
-    def create(
-        cls,
-        data,
-        *args,
-        **kwargs,
-    ):
-        record = super().create(data, **kwargs)
-        record.update_refs_in_citation_table()
-        return record
-
-    def update(
-        self,
-        data,
-        *args,
-        **kwargs,
-    ):
-        super().update(data)
+    def update_record_relationships(self):
```

Comment:
@DonHaul could you please check this https://github.com/cern-sis/issues-inspire/issues/598